analyzers/src/SonarAnalyzer.CFG/ShimLayer/AnalysisContext/SymbolStartAnalysisContextWrapper.cs

```diff
@@ -132,10 +206,8 @@ static SymbolStartAnalysisContextWrapper()
         }
         else if (languageKindType.FullName == VBSyntaxKind)
         {
-            // See https://github.com/SonarSource/sonar-dotnet/pull/9028 for how to implement this
-            // Attention: Do not add a package reference to "Microsoft.CodeAnalysis.VisualBasic.Workspaces".
-            // It creates hard to detect file not found errors in .NET SDK 3 and .NET SDK 5 and other scenarios
-            // we do not fully understand.
+            Action<object> wrapper = x => action((CodeBlockStartAnalysisContext<TLanguageKindEnum>)x);
+            RegisterCodeBlockStartActionVB(RoslynSymbolStartAnalysisContext, wrapper);
```

Comment:
I would embed this one. And also the one above for C#
```suggestion
            RegisterCodeBlockStartActionVB(RoslynSymbolStartAnalysisContext, x => action((CodeBlockStartAnalysisContext<TLanguageKindEnum>)x));
```